lms/djangoapps/support/static/support/jsx/entitlements/components/EntitlementForm/index.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, InputSelect, InputText, TextArea } from '@edx/paragon';
+
+
+class EntitlementForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const isReissue = props.entitlement !== null && props.entitlement !== undefined;
+
+    this.state = {
+      isReissue,
+      courseUuid: isReissue ? props.entitlement.courseUuid : '',
+      mode: isReissue ? props.entitlement.mode : '',
+      username: isReissue ? props.entitlement.user : '',
+      comments: 'Add any additional comments here',
+    };
+
+
+    this.handleCourseUUIDChange = this.handleCourseUUIDChange.bind(this);
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+    this.handleModeChange = this.handleModeChange.bind(this);
+    this.handleCommentsChange = this.handleCommentsChange.bind(this);
+    this.submitForm = this.submitForm.bind(this);
+    this.onClose = this.onClose.bind(this);
```

Comment:
This is pretty nitpicky so feel free to disregard, but for readability it would be nice to have to order of function bindings be the same as the function definitions (the only one that's out of place is `onClose` which is bound last but defined first)